llvm/include/llvm/TableGen/StringToOffsetTable.h

```diff
@@ -52,6 +52,33 @@ class StringToOffsetTable {
     return II->second;
   }
 
+  // Emit the string using string literal concatenation, for better readability
+  // and searchability.
+  void EmitStringLiteralDef(raw_ostream &OS, const Twine &Decl,
+                            const Twine &Indent = "  ") const {
+    OS << "\n"
```

Comment:
(nit) could use a raw literal here

Reply:
Done (and need to use formatv for indent as well).